dev/benchmarks/complex_layout/lib/main.dart

```diff
@@ -7,13 +7,15 @@ import 'package:flutter/scheduler.dart' show timeDilation;
 
```

Comment:
do these changes affect benchmark numbers?

Reply:
Will keep an eye on the numbers.